tests/voice-isolate-processor.test.js

```
@@ -0,0 +1,298 @@
+    expect(proc.hopsSinceInit).toBe(0);
+  });
+
+  test('initRingBuffers: posts ready message', () => {
+    proc._onMessage({ type: 'initRingBuffers', fftSize: 4096, hopSize: 1024 });
+    expect(proc.port.postMessage).toHaveBeenCalledWith(
+      expect.objectContaining({ type: 'ready', fftSize: 4096 })
+    );
+  });
+
+  test('initRingBuffers: posts error when inputSAB is too small', () => {
+    const tinyBuf = new SharedArrayBuffer ? (() => {
```

ESLint

[error] Unexpected constant condition.

(no-constant-condition)